docs/index.rst

```diff
@@ -15,7 +15,7 @@ For more details, see `GitHub. <https://github.com/ROCm/rocminfo>`_
 
   .. grid-item-card:: Install
```

Comment:
.. grid-item-card:: Install

Should this line be:
.. grid-item-card:: Build

Reply:
Fixed. Thanks, David.